src/TestContext.elm

```diff
@@ -425,6 +468,133 @@ clickButton buttonText testContext =
         testContext
 
 
+{-| Simulates replacing the text in an input field labeled with the given label.
+
+NOTE: Currently, this function requires that you also provide the field id
+(which must match both the `id` attribute of the target `input` element,
+and the `for` attribute of the `label` element).
+After [eeue56/elm-html-test#52](https://github.com/eeue56/elm-html-test/issues/52) is resolved,
+a future release of this package will remove the `fieldId` parameter.
+
+NOTE: TODO: In the future, this will be generalized to work with
+labeled textareas as well as labeled input fields.
+(related to [eeue56/elm-html-test#49](https://github.com/eeue56/elm-html-test/issues/49>))
+
+NOTE: In the future, this will be generalized to work with
+aria accessiblity attributes in addition to working with standard HTML label elements.
+
+If you need to target a `<textarea>` that does not have a label,
+see [`fillInTextarea`](#fillInTextArea).
+
+If you need more control over the finding the target element or creating the simulated event,
+see [`simulate`](#simulate).
+
+-}
+fillIn : String -> String -> String -> TestContext msg model effect -> TestContext msg model effect
+fillIn fieldId label newContent testContext =
+    simulateLabeledInputHelper ("fillIn " ++ toString label)
```

Comment:
why `toString` here? Is it just to get the quotes? It seems better to manually do that formatting since `toString` will go away soon.

Reply:
I think it's ideal in that it also gives backslash escaping for quotes and other special characters.

Reply:
ohhh, good point!